pkg/component/ai/instill/v0/embedding.go

```diff
@@ -0,0 +1,37 @@
+package instill
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	modelPB "github.com/instill-ai/protogen-go/model/model/v1alpha"
```

Comment:
nit: go packages aren't capitalised and aliases should follow that convention
```suggestion
	modelpb "github.com/instill-ai/protogen-go/model/model/v1alpha"
```